wdl/imputation.wdl

```diff
@@ -308,25 +785,30 @@ task batch_qc {
     String dollar = "$"
     String docker
 
-
     command <<<
-        set -euo pipefail
+
+        set -euxo pipefail
         mem=$((`free -m | grep -oP '\d+' | head -n 1`-500))
         plink_cmd="plink --allow-extra-chr --keep-allele-order --memory $mem"
         plink2_cmd="plink2 --allow-extra-chr --memory $mem"
 
         echo "get variants to exclude based on joint qc and panel comparison"
         for file in ${sep=" " exclude_variants_joint}; do cat $file >> upfront_exclude_variants.txt; done
-        cat ${exclude_variants_panel} >> upfront_exclude_variants.txt
+        if [ "${chip_qc}" = "true" ]; then
+            echo "chip qc: not excluding variants that are not in panel or have low frequency in panel"
+            cat ${exclude_variants_panel} | grep -Ev "not_in_panel|af_panel" >> upfront_exclude_variants.txt
```

Comment:
Totally agree, was thinking the same, possibly one option would be to give these strings as inputs, but well. We do get the summary plots of excluded variants and will spot from there if something goes awry. I add some comments about this so that no one will change af_panel to panel_af:)

Reply:
Actually I won't add comments now because exactly not_in_panel and af_panel need to be reincluded for imp QC